portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Policies/components/TabPanel.tsx

```diff
@@ -45,27 +45,14 @@ const TabPanel: FC<TabPanelProps> = ({
     const currentFlow = flowNames[index];
 
     return (
-        <div
-            role='tabpanel'
-            hidden={selectedTab !== index}
-            id={`${currentFlow}-tabpanel`}
-            aria-labelledby={`${currentFlow}-tab`}
-        >
-            <Box py={1} px={3}>
-                {selectedTab === index &&
-                    policyList?.map((singlePolicy: Policy) => {
-                        return (
-                            <DraggablePolicyCard
-                                key={singlePolicy.id}
-                                policyObj={singlePolicy}
-                                showCopyIcon
-                                isLocalToAPI={singlePolicy.isAPISpecific}
-                                fetchPolicies={fetchPolicies}
-                            />
-                        );
-                    })}
-            </Box>
-        </div>
+        <TabPanelShared
+            selectedTab={selectedTab}
+            index={index}
+            currentFlow={currentFlow}
+            policyList={policyList}
+            fetchPolicies={fetchPolicies}
+            DraggablePolicyCard={DraggablePolicyCard}
```

Comment:
I have used a FlowArrow.tsx shared file in the shared folder and reused them in both features. Eventhough it has only direction as a data (Which is not a problem for resuing in both features) I kept the file structure as it is for the readability. Becase if so, the only file missing will be that and I thought it will confuse people. If we needed, we can completely remove it from both features as it's in the bottom of the react components of both features.